cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,42 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ArticlePageObject from '../support/pages/article.pageObject';
+
+const articlePage = new ArticlePageObject();
+
 describe('Article', () => {
   before(() => {
+  let article;
```

Comment:
The `article` variable is declared here but not initialized. Ensure that the `generateArticle` task successfully assigns a value to it before it is used in tests.